requirements/requirements-avro.txt

```
@@ -1,3 +1,3 @@
 fastavro>=0.23.0
```

There was a problem with fastavro and 3.7 in versions higher than 1.8.0, but maybe it's been fixed in 1.8.1
```suggestion
fastavro < 1.8.0; python_version == "3.7"
fastavro < 2; python_version > "3.7"
```